src/api/app/controllers/webui/cloud/upload_jobs_controller.rb

```diff
@@ -3,13 +3,20 @@ module Cloud
     class UploadJobsController < WebuiController
       before_action :require_login
       before_action -> { feature_active?(:cloud_upload) }
-      before_action :validate_configuration_presence, :set_breadcrump
+      before_action :validate_ec2_configuration_presence, only: [:new, :create]
+      before_action :set_breadcrump
       before_action :validate_uploadable, only: [:new]
       before_action :set_package, only: :new
       before_action :set_upload_job, only: :destroy
 
       def index
+        unless User.current.cloud_configurations?
+          redirect_to configuration_cloud_path
```

Comment:
The helper looks wrong, it should be ``cloud_configurations_path``. Maybe you do something wrong in the ``routes.rb``

Reply:
I actually didn't manage to rename this to make it consistent to the other changes. Maybe you can help me in this regard.

Reply:
I would use resource in the routes.rb file which should do the trick. 
The problem is that you use ``get``, if you use get you also need to use ``as: ''`` IIRC. But as said, I prefer using resource!